engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -359,6 +360,11 @@ public void onChunkUnloaded(Vector3i pos) {
     }
 
     @Override
+    public void requestRenderTaskListRefresh() {
+        requestedRenderTaskListRefresh = true;
```

Comment:
In line with comment above this variable can just be `requestedTaskListRefresh`.